### ISSUE TITLE
[Analytics: News] Track page view

### ISSUE THREAD

**Issue by pahupe** [urfuwo/hlx-test#206]
Preconditions: see tickets #103 and #102

Goals:
* Populate the data layer with actual site & page data of the current page (see data layer structure below)
* Page views show in Analytics report (end-to-end)

Constraints: 
* [More details from EPAM needed] Populate the data layer following SAP’s multi-phase DL population approach: Three events populate the DL:
    * globalDL
    * pageView
    * stBeaconReady
* Data is sent to Analytics only once, not for every event (to limit Analytics calls)

SAP DL structure (references see below):
page:
    * country (“glo” = global site) // more details needed 
    * language // assume “en”
    * name
    * referrer
    * section
    * title (w/slug)
    * url
site:
    * country <= see page data
    * name // assume “news”
user (shall be hard-coded in this ticket):
    * loginStatus // assume “no”
    * type // assume “visitor”

Please crosscheck attributes against the Wiki reference page "SAP EDDL: Global, Page and User spec" (reference below).

Not in scope:
* actual user status / login status, will be covered in a separate ticket

References:
* Existing implementation: https://news.sap.com/blog/ => console: adobeDataLayer.getState()
* SAP EDDL: Global, Page and User spec: https://wiki.one.int.sap/wiki/display/DBA/EDDL+v.1.0.0.0+RC+-+Global+and+Page

**Comment by pahupe** [comment]
To be validated with Sankalp Singh

**Comment by pahupe** [comment]
This is done and live on www-qa.sap.com
